plugin/trino-password-authenticators/src/main/java/io/trino/plugin/password/ldap/LdapUtil.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.password.ldap;
+
+import com.google.common.base.CharMatcher;
+
+public final class LdapUtil
```

Comment:
Moving to Util clas can be in a separate commit

Reply:
I still don't get that rule. `LdapUtil` was introduced as well as other classes in the first commit. Why it requires a separate commit?